src/CTA.Rules.Actions/ClassActions.cs

```diff
@@ -152,5 +152,23 @@ public class ClassActions
             };
             return RenameClass;
         }
+        public Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> GetReplaceMethodModifiersAction(string methodName, string modifiers)
+        {
+            Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> ReplaceMethodModifiers = (SyntaxGenerator syntaxGenerator, ClassDeclarationSyntax node) =>
+            {
+                var allMembers = node.Members.ToList();
+                var allMethods = allMembers.OfType<MethodDeclarationSyntax>();
+                var replaceMethod = allMethods.Where(m => m.Identifier.ToString() == methodName).FirstOrDefault();
+                allMembers.Remove(replaceMethod);
```

Comment:
This will probably fail if it runs on an empty class

Reply:
Sorry, good catch I fixed it and also fixed RemoveMethod action which this was based on.